sdk/typescript/src/utils/errors.ts

```diff
@@ -19,7 +19,12 @@ export class RPCError extends Error {
     data?: unknown;
     cause?: Error;
   }) {
-    super('RPC Error', { cause: options.cause });
+    super(
+      options.cause
+        ? `RPC Error: ${options.cause.message}`
```

Comment:
Making this change just to help improve debugging for platforms that don't display error cause